### ISSUE TITLE
No handlers could be found for logger "pychromecast.socket_client"

### ISSUE THREAD

**Issue by dhdurgee** [muammar/mkchromecast#120]
I am encountering cases where the stream stops playing with the above message.  Here is the console:

dhdurgee@Z560 ~/Downloads $ mkchromecast --source-url http://17413.live.streamtheworld.com:80/WWRCAMAAC_SC
Mkchromecast v0.3.7
 
List of Devices Available in your Network:
------------------------------------------

Index 	Types 	Friendly Name 
===== 	===== 	============= 
0 	Gcast 	WD32FB2530
 
We will cast to first device in the list above!
 
WD32FB2530
 
 
Information about  WD32FB2530
 
DeviceStatus(friendly_name=u'WD32FB2530', model_name=u'Chromecast', manufacturer=u'Google Inc.', api_version=(1, 0), uuid=UUID('b59b8971-64c6-46d3-118b-b99e9264a21a'), cast_type='cast')
 
Status of device  WD32FB2530
 
CastStatus(is_active_input=False, is_stand_by=True, volume_level=1.0, volume_muted=False, app_id=u'E8C28D3C', display_name=u'Backdrop', namespaces=[u'urn:x-cast:com.google.cast.debugoverlay', u'urn:x-cast:com.google.cast.cac', u'urn:x-cast:com.google.cast.sse'], session_id=u'd3d5b203-4bd6-4ecf-9d19-426a7b780ada', transport_id=u'd3d5b203-4bd6-4ecf-9d19-426a7b780ada', status_text=u'')
 
The IP of WD32FB2530 is: 192.168.230.2
Your local IP is: 192.168.230.233
 
Casting from stream URL: http://17413.live.streamtheworld.com:80/WWRCAMAAC_SC
Using media type: audio/mpeg
 
Cast media controller status
 
CastStatus(is_active_input=False, is_stand_by=True, volume_level=1.0, volume_muted=False, app_id=u'CC1AD845', display_name=u'Default Media Receiver', namespaces=[u'urn:x-cast:com.google.cast.debugoverlay', u'urn:x-cast:com.google.cast.broadcast', u'urn:x-cast:com.google.cast.media'], session_id=u'7b43fe33-d99a-4692-bc7e-4b4eb58b7704', transport_id=u'7b43fe33-d99a-4692-bc7e-4b4eb58b7704', status_text=u'Ready To Cast')
 
Remember to open pavucontrol and select the mkchromecast sink.

Ctrl-C to kill the Application at any Time

No handlers could be found for logger "pychromecast.socket_client"
^CKilled
dhdurgee@Z560 ~/Downloads $ 

When the error occurs the player stops on the TV and all I can do is kill the current task and start again.  Is there more information I can provide?  Are there additional options I need to use with the command line?

**Comment by muammar** [comment]
Do you still get the problem?

**Comment by dhdurgee** [comment]
Yes, I do.  I get this "no handlers for logger" occasionally.  Sometimes with a halt of the stream, sometimes not.  I also run into cases where the stream halts without the message.  From the message itself I assume that somef condition is occurring that pychromecast wants to report, but the logger is not ready to accept it.
